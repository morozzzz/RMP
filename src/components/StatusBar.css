.status-bar {
    background-color: rgb(238, 238, 238);
    font-weight: bold;
}

.status-bar-content {
    min-height: 2.5em;
    display: flex;
    justify-content: space-between;
    align-items: center;
    width: 60%;
    margin: 0 auto;
}