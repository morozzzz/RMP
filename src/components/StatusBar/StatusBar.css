.status-bar {
    background-color: rgb(238, 238, 238);
    font-weight: bold;
    height: 2.5em;
}

.status-bar__content {
    height: 100%;
    display: flex;
    justify-content: space-between;
    align-items: center;
    width: 60%;
    margin: 0 auto;
}