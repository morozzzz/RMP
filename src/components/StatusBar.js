import React from 'react';
import './StatusBar.css';

const StatusBar = ({ status, children }) => (
    <div className="status-bar">
        <div className="status-bar-content">
            <div className="list-status">{status}</div>
            {children}
        </div>
    </div>
);

export default StatusBar;
