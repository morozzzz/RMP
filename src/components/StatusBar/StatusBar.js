import React from 'react';
import './StatusBar.css';

const StatusBar = ({ status, children }) => (
    <div className="status-bar">
        <div className="status-bar__content">
            <div className="status-bar__list-status">{status}</div>
            {children}
        </div>
    </div>
);

export default StatusBar;
