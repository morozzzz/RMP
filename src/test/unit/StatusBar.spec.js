import React from 'react';
import { shallow } from 'enzyme';
import StatusBar from '../../components/StatusBar/StatusBar';

describe.skip('StatusBar', () => {
    it('should be rendered correctly', () => {
        const statusMock = 'some status';
        const wrapper = shallow(<StatusBar status={statusMock} />);

        expect(wrapper).toMatchSnapshot();
    });
});
