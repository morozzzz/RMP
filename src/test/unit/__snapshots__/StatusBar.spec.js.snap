// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar should be rendered correctly 1`] = `
<div
  className="status-bar"
>
  <div
    className="status-bar__content"
  >
    <div
      className="status-bar__list-status"
    >
      some status
    </div>
  </div>
</div>
`;
